Content.Server/AlertLevel/AlertLevelOnPressComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.AlertLevelOnPress;
+using Robust.Shared.Audio;
+
+namespace Content.Server.AlertLevelOnPress;
```

Comment:
move this to shared and remove the empty SharedAlertLevelOnPress

make abstract SharedAlertLevelOnPressSystem for access

Reply:
only issue with this is accessing the ProtoId<AlertLevelPrototype> from shared